crux-llvm/install.project

```diff
@@ -0,0 +1,11 @@
+packages:
```

Comment:
This `install.project` file is no longer used

Reply:
Fixed.